### DISCUSSION TITLE
compatible file format for Mac

### DISCUSSION THREAD

**Discussion by bradfc** [dnschneid/crouton#4678] [Q&A] [answered]
I'm running a new Lenovo Flex 5 with Intel i3-1115G4 and have installed Ubuntu 20.04+MATE. Per the guidance for Intel processors.  I would like to use the Lenovo while traveling to copy raw image photos from an SD card formatted in exFAT to an external SSD.  I tried formatting the SDD as exFAT, and have given various commands (say, rsync) to copy the .CR2 files from the SD card to the SSD, but when I mount the SSD on a Mac, it sees only a fraction of the files.

As an aside, I also tried using the ChromeOS 'finder' windows and dragging the icons from one drive to the other, but it wasn't very smart: it created "(1)" copies instead of overwriting or ignoring files that were already there.

Is there a way to copy files across external drives with a format that is readable by a Mac?

**Comment by TheTechRobo** [comment]
how many photos?

**Reply by bradfc** [reply]
About 25 or so (about 20 MB each).  The 500 GB SSD drive is essentially empty.

What happens is that the system enumerates each file as copied, but in fact some are not.

**Reply by bradfc** [reply]
addendum: it may be a write cache issue: if I wait longer, more files are transferred.

I'm happy to wait, but 

- I can't find a way to tell if the transfer is complete on the Chromebook.  Both 'ls' and FileManager show all the files as transferred, but only a fraction of them are there when I view the drive on a Mac.
- I can't find a way to force the process by ejecting the drive because mounting it in the first place was outside my control: as soon as I plugged the drive I got a message that it was "already mounted"

**Reply by dnschneid** [reply]
Have you tried running `sync` when the copy command is done?  It should take a bit of time to complete as the files are getting physically written to disk.

**Reply by bradfc** [reply]
It appears to work!

I tried it twice (erased external drive, ran rsync, then "sudo sync" just to sync everything), and all files are there when I attach to a Mac.

Many thanks!!

**Reply by dnschneid** [reply]
There's no need to use `sudo` for `sync`, but glad that solved the problem.

**Comment by dnschneid** [comment, marked as answer]
> Run `sync` when the copy command is done. It should take a bit of time to complete as the files are getting physically written to disk.
